wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -896,12 +909,80 @@ int atcatls_verify_signature_cb(WOLFSSL* ssl, const byte* sig, unsigned int sigS
     return ret;
 }
 
+static int atcatls_set_certificates(WOLFSSL_CTX *ctx) 
+{
+    int ret = 0;
+    ATCA_STATUS status;
+    size_t signerCertSize = 0;
+    uint8_t signerCert[signerCertSize];
+    size_t deviceCertSize = 0;
+    uint8_t deviceCert[deviceCertSize];
+    int devPemSz, signerPemSz;
+    byte devPem[1024];
+    byte signerPem[1024];
+    char devCertChain[devPemSz+signerPemSz];
+
+    /*Read signer cert*/
+    status = tng_atcacert_max_signer_cert_size(&signerCertSize);
+    if (ATCA_SUCCESS != status) {
+        ret = atmel_ecc_translate_err(ret);
+        return ret;
+    }
+    status = tng_atcacert_read_signer_cert((uint8_t*)&signerCert, &signerCertSize);
+    if (ATCA_SUCCESS != status) {
+        ret = atmel_ecc_translate_err(ret);
+        return ret;
+    }
+    /*Read device cert signed by the signer above*/
+    status = tng_atcacert_max_device_cert_size(&deviceCertSize);
+    if (ATCA_SUCCESS != status) {
+        ret = atmel_ecc_translate_err(ret);
+        return ret;
+    }
+    status = tng_atcacert_read_device_cert((uint8_t*) & deviceCert, &deviceCertSize, (uint8_t*) & signerCert);
```

Comment:
Please cleanup the dereferences to remove spaces. Also if you are passing in a byte/char array you don't need the dereference `&`.